lib/util/markdown.ts

```diff
@@ -6,7 +6,7 @@ import { regEx } from './regex';
 export function sanitizeMarkdown(markdown: string): string {
   let res = markdown;
   // Put a zero width space after every # followed by a digit
-  res = res.replace(regEx(/#(\d)/gi), '#&#8203;$1');
+  res = res.replace(regEx(/\W#(\d)/gi), '#&#8203;$1');
```

Comment:
this replaces the char before the `#` 🤦‍♂️

Reply:
Do you have an example....

Reply:
Would this be better

regex: `(\W)#(\d)`
replacement: $1#&#8203;$2

Reply:
The other route I explored was using `%23` instead of the `#` in `release-notes.ts` and then re-replacing it from `%23` to `#` at the end of `sanitizeMarkdown`....

Reply:
I tried reproducing the "removes the char before #", but couldn't... 🤔

Reply:
> I think _this_ might be better
> 
> regex: `(\W)#(\d)`
> replacement: $1#&#8203;$2

yes, otherwise the chat before will be removed, see screenshot and sample markdown on other PR comment